implementation/src/main/java/io/smallrye/metrics/MetricsRequestHandler.java

```diff
@@ -102,9 +105,10 @@ public void handleRequest(String requestPath,
         }
 
         if (!requestPath.startsWith(contextRoot)) {
-            responder.respondWith(500, "The expected context root of metrics is "
-                    + contextRoot + ", but a request with a different path was routed to MetricsRequestHandler",
-                    Collections.emptyMap());
+            String message = "The expected context root of metrics is "
+                    + contextRoot + ", but a request with a different path was routed to MetricsRequestHandler";
```

Comment:
I would use quotation marks for the context root e.g.:
`
String message = "The expected context root of metrics is '" + contextRoot + "', but a request with a different path was routed to MetricsRequestHandler";
'
This makes it easier to spot white spaces in the string. Again. this is just convenience or habit of mine as there should be no white spaces in a context root...

Reply:
Ok why not.. I added quotes